lib/aytests/image_builder.rb

```diff
@@ -92,7 +93,7 @@ def upgrade(autoinst, iso_url)
       setup_definition(:upgrade)
       change_boot_order
       backup_image
-      build
+      build(build_environment(autoinst_path.sub_ext(".linuxrc")))
```

Comment:
Well, actually you need to pass `autoinst`, which I see fine because is quite simetric to the call to `setup_autoinst`.